src/components/SettingsDialog/SettingsDialog.vue

```diff
@@ -43,14 +43,15 @@
 		<AppSettingsSection v-if="!isGuest"
 			:title="t('spreed', 'Privacy')"
 			class="app-settings-section">
-			<input id="read_status_privacy"
+			<CheckboxRadioSwitch id="read_status_privacy"
 				:checked="readStatusPrivacyIsPublic"
 				:disabled="privacyLoading"
 				type="checkbox"
 				name="read_status_privacy"
 				class="checkbox"
-				@change="toggleReadStatusPrivacy">
-			<label for="read_status_privacy">{{ t('spreed', 'Share my read-status and show the read-status of others') }}</label>
+				@update:checked="toggleReadStatusPrivacy">
+        {{ t('spreed', 'Share my read-status and show the read-status of others') }}
```

Comment:
src/components/SettingsDialog/SettingsDialog.vue#L53

Expected "\t" character, but found " " character

```suggestion
			{{ t('spreed', 'Share my read-status and show the read-status of others') }}
```